### ISSUE TITLE
Mounted partitions are shown as removable devices even when they are mounted with fstab

### ISSUE THREAD

**Issue by woox2k** [mate-desktop/caja#158]
Hello there.
This issue may be intended feature but then it's feature request.
My problem is that no matter how or where (except / and /mnt) i mount my internal drive partitions they appear as removable devices under "Places" and i can unmount them accidentally even if there's programs running off it and etc.

I would understand if my drives were mounted with a mount command or into /media or /run directory, they are mounted with fstab and into /home/disk... and still show up as removable devices.

Long story short:
Can you fix the issue where non /media, /run mounts are shown as removable devices
or make caja read fstab to see what paritions are included there and exclude those from the removable device list. (I doubt that someone uses fstab to mount USB sticks that are not on the machine 24/7...)

Caja version: 1.6.2-1
OS: Archlinux x64/Linux Mint 15
